float32 Sx
float32 Sy
float32 Syaw
---
visualization_msgs/Marker[] lanelets
